src/Lecture2.hs

```diff
@@ -97,7 +109,10 @@ spaces.
 
 🕯 HINT: look into Data.Char and Prelude modules for functions you may use.
 -}
-dropSpaces = error "TODO"
+
+dropSpaces :: [Char] -> [Char]
+dropSpaces string = dropWhile space (reverse (dropWhile space (reverse string)))
+  where space char = char == ' '
```

Comment:
Alternatively, you can import the `isSpace` function from the `Data.Char` module to avoid defining your own function 👾